.github/workflows/ci.yaml

```diff
@@ -0,0 +1,25 @@
+name: Component Graph CI
+
+on:
+  push:
+    branches: [ v0.x.x ]
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
I would be on the conservative side and pin this to some concrete ubuntu version.

Reply:
Pinned to 20.04.  Still have 6 months before that reaches end of life. \o/

Reply:
Why not 22.04 or even 24.04 then?